### PR TITLE
Support property condition: `simple`

### LINKED ISSUE #54: Implement property condition: Simple
__What should we do?__

* I can set property condition of `"type": "simple"`
* I can set property `id`

__Why should we do it?__

Part of https://github.com/bpmn-io/bpmn-js-properties-panel/issues/690

### PR DESCRIPTION
**Pull request by barmac**
```json
"condition": {
  "examples": [
    {
      "type": "simple",
      "property": "httpMethod",
      "equals": "GET"
    },
    {
      "type": "simple",
      "property": "httpMethod",
      "oneOf": [ "POST", "PUT", "DELETE" ]
    }
  ]
}
```

Closes #54

### CODE REVIEW

**Comment by barmac** [comment]
Note for reviewers: This PR is targeted at `next`. This is the branch from which we will release an alpha version of the schema. If we need to do any breaking changes, we will do it on `next`. When condition schema is settled, we will merge to `master`.

**Comment by nikku** [comment]
Should we not adopt `develop` and `master/main`? We have it in some parts of our tooling, and it works.

**Comment by barmac** [comment]
We could do that, sure.

**Comment by barmac** [comment]
`next` -> `develop` is done

File: packages/element-templates-json-schema-shared/src/defs/condition.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  ],
+  "errorMessage": {
+    "required": {
+      "property": "missing property name for condition \"${0#}\""
+    }
+  },
+  "properties": {
+    "type": {
+      "$id": "#/condition/type",
+      "const": "simple",
```
Comment (pinussilvestrus) [inline packages/element-templates-json-schema-shared/src/defs/condition.json:21]: If we are that explicit, should we have a test for this? `type !== simple`.

**Review by pinussilvestrus — COMMENTED** [review]
Looks good to me, just some minimum remarks/questions 👍

File: packages/zeebe-element-templates-json-schema/test/fixtures/condition-missing-property.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          keyword: 'required',
+          dataPath: '/properties/1/condition',
+          schemaPath: '#/definitions/properties/allOf/0/items/properties/condition/required',
+          params: { missingProperty: 'property' },
+          message: "should have required property 'property'",
+          emUsed: true
+        }
+      ]
+    },
+    message: 'missing property name for condition ""condition""'
```
[the remark this answers is not in the document]
Comment (barmac) [reply packages/zeebe-element-templates-json-schema/test/fixtures/condition-missing-property.js:49]: I think I overlooked this. I will remove the quotes.

File: packages/zeebe-element-templates-json-schema/resources/schema.json [hunk 3]
```diff
@@ -164,6 +168,84 @@
...
+                  "required": {
+                    "property": "missing property name for condition \"${0#}\""
+                  }
+                },
+                "properties": {
+                  "type": {
+                    "$id": "#/condition/type",
+                    "const": "simple",
+                    "description": "The type of the condition.",
+                    "default": "simple"
```
[the remark this answers is not in the document]
Comment (barmac) [reply packages/zeebe-element-templates-json-schema/resources/schema.json:209]: I think this means we should also have a test case for missing keywords when type is not provided.

File: packages/element-templates-json-schema-shared/src/defs/condition.json (hunk 1 above)
Comment (barmac) [reply packages/element-templates-json-schema-shared/src/defs/condition.json:21]: I will add this.

**Comment by barmac** [comment]
Thanks for the review. I will add change you suggested and the test cases.

**Comment by barmac** [comment]
Ready for second round.

**Review by pinussilvestrus — APPROVED** [review]
LGTM 👍  let's try this out!

**Pull request merged**